src/Core/src/Platform/iOS/SwitchExtensions.cs

```diff
@@ -40,20 +47,17 @@ public static void UpdateThumbColor(this UISwitch uiSwitch, ISwitch view)
 				uiSwitch.ThumbTintColor = thumbColor?.ToPlatform();
 		}
 
-		internal static UIView GetTrackSubview(this UISwitch uISwitch)
+		internal static UIView? GetTrackSubview(this UISwitch uISwitch)
 		{
-			UIView uIView;
 			if (OperatingSystem.IsIOSVersionAtLeast(13) || OperatingSystem.IsTvOSVersionAtLeast(13))
-				uIView = uISwitch.Subviews[0].Subviews[0];
+				return uISwitch.Subviews?[0]?.Subviews?[0];
 			else
-				uIView = uISwitch.Subviews[0].Subviews[0].Subviews[0];
-
-			return uIView;
+				return uISwitch.Subviews?[0]?.Subviews?[0]?.Subviews?[0];
 		}
```

Comment:
Do these views have names or something? Also, what happens if there are no subviews - but it is not null, will the `[0]` throw an index out of bounds?

Reply:
The views do no have any names, whips is annoying. But we can work around it, @tj-devel709 we can use a FirstOrDefault from Line to solve what @mattleibow mentioned. FirstOrDefault in an empty collection will be null, so we do null null null

Reply:
@mandel-macaque are we concerned about the overhead of using LINQ here at all?

Reply:
LINQ is indeed not a good idea. We can always add a helper method like this:
```csharp
public static T FirstOrDefaultNoLinq<T> (T[] items)
{
	return items switch {
		[var i, ..] => i,
		[] => default,
	};
}
```
Does not use LINQ, is a little evil, but should work.